vision/google/cloud/vision/geometry.py

```diff
@@ -87,6 +101,19 @@ def from_api_repr(cls, response):
         longitude = response['latLng']['longitude']
         return cls(latitude, longitude)
 
+    @classmethod
+    def from_pb(cls, response):
+        """Factory: construct location information from Vision gRPC response.
+
+        :type response: :class:`~google.cloud.vision.v1.LocationInfo`
+        :param response: gRPC response of ``LocationInfo``.
+
+        :rtype: :class:`~google.cloud.vision.geometry.LocationInformation`
+        :returns: ``LocationInformation`` with populated latitude and
+                  longitude.
+        """
+        return cls(response.lat_lng.latitude, response.lat_lng.longitude)
```

Comment:
Doesn't appear to be...
https://github.com/googleapis/api-client-staging/blob/22a53426ff8308f4bc227fa3aa529b63ff3e63cf/generated/python/grpc-google-cloud-vision-v1/google/cloud/grpc/vision/v1/image_annotator_pb2.py#L564